shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -557,8 +558,16 @@ public void execute(final String[] args) throws IOException {
     }
   }
 
+  public static class NonEscapeParser extends org.jline.reader.impl.DefaultParser {
+    @Override
+    public boolean isEscapeChar(char ch) {
```

Comment:
We should verify that only the backslash char is affected by this override. 

Otherwise disabling all escaping via the option in the builder might be cleaner as the option name is referenced in jline documentation. 

 `reader.setOption(LineReader.Option.DISABLE_EVENT_EXPANSION, true);`